test/test_helper.rb

```diff
@@ -14,6 +14,7 @@ class ActiveSupport::TestCase
   end
 
   def setup
+    SimpleCov.start 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.